lib/internal/Magento/Framework/File/Test/Unit/Transfer/Adapter/HttpTest.php

```diff
@@ -3,12 +3,21 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+
 namespace Magento\Framework\File\Test\Unit\Transfer\Adapter;
 
 use \Magento\Framework\File\Transfer\Adapter\Http;
 
+/**
+ * Tests http transfer adapter.
+ */
 class HttpTest extends \PHPUnit\Framework\TestCase
 {
+    /**
+     * @var \Magento\Framework\App\Request\Http|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
Use `PhpUnit\Framework\MockObject` instead

Reply:
Would you like me to also change the rest of the test to use `PhpUnit\Framework\MockObjec`t instead?

Reply:
It's better to use the one namespace import for the same class instead of two. Please, replace other usages in this test.